js/src/main/scala/cats/jvm/std/future.scala

```diff
@@ -0,0 +1,50 @@
+package cats
+package js
+package std
+
+import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext => E}
+import scala.concurrent.duration.FiniteDuration
+
+import cats.std.FutureCoflatMap
+import cats.syntax.all._
+
+object future extends FutureInstances0
+
+object Await {
+  def result[A](f: Future[A], atMost: FiniteDuration): A = f.value match {
+      case Some(v) => v.get
+      case None => throw new IllegalStateException()
+    }
+}
+
+trait FutureInstances0 extends FutureInstances1 {
+  def futureComonad(atMost: FiniteDuration)(implicit ec: E): Comonad[Future] =
+    new FutureCoflatMap with Comonad[Future] {
+      def extract[A](x: Future[A]): A =
+        Await.result(x, atMost)
+    }
+
+  def futureOrder[A: Order](atMost: FiniteDuration)(implicit ec: E): Eq[Future[A]] =
```

Comment:
I've also noticed that the dir is wrong (has jvm in path). 

`js/src/main/scala/cats/jvm/std/future.scala` to `js/src/main/scala/cats/js/std/future.scala`